src/AppBundle/Donation/DonationRequest.php

```diff
@@ -88,6 +88,11 @@ class DonationRequest
      */
     private $phone;
 
+    /**
+     * @Assert\NotBlank()
```

Comment:
Extra parenthesis can be removed.
Thus you should validate the value against a `Choice` constraint with a whitelist of choices.